### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by Ichbinjoe**
Basic issue template for the different types of issues we encounter. Hopefully, this will weed out some not-so quality issues.

### CODE REVIEW

**Comment by willies952002** [comment]
:+1:

**Comment by zachbr** [comment]
What does github do with the contributing file that it's advantageous to put it there?

**Comment by kashike** [comment]
@Zbob750 https://help.github.com/articles/setting-guidelines-for-repository-contributors/
@Ichbinjoe Rename `Contributing.md` to `CONTRIBUTING.md`.

**Comment by zachbr** [comment]
Thanks :)

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### Plugin list:
+
+### bukkit.yml, spigot.yml, paper.yml, server.properties
+___Gist/pastebin/hastebin links___
+### Other helpful links
+___The more information we receive, the quicker and more effective we can be at finding the solution to the
+issue.___
+
+### Paper build number:
+___This can be found by running `/version` on your server___
```
Comment (zachbr) [inline .github/ISSUE_TEMPLATE.md]: Can we make it explicitly say "Latest is not a version and will not be accepted" here and the other places? Even the source code builds will output a version string with the commit hash, there's really no excuse /rant

**Comment by Ichbinjoe** [comment]
I have made those changes.

**Comment by kashike** [comment]
@Ichbinjoe 

> @Ichbinjoe Rename Contributing.md to CONTRIBUTING.md.

**Comment by Ichbinjoe** [comment]
Forgot to add the change. Fixed.

**Pull request merged**